server/api/admin.go

```diff
@@ -66,6 +66,11 @@ func (h *adminHandler) ResetTS(w http.ResponseWriter, r *http.Request) {
 	}
 
 	if err := handler.ResetTS(uint64(ts)); err != nil {
-		h.rd.JSON(w, http.StatusInternalServerError, err.Error())
+		if err == server.ErrServerNotStarted {
+			h.rd.JSON(w, http.StatusInternalServerError, err.Error())
+		} else {
+			h.rd.JSON(w, http.StatusForbidden, err.Error())
```

Comment:
How about `BadRequest`?

Reply:
```
400 is the generic client-side error status, used when no other 4xx error code is appropriate. Errors can be like malformed request syntax, invalid request message parameters, or deceptive request routing etc.
403 error response indicates that the client’s request is formed correctly, but the REST API refuses to honor it .
```
I think `BadRequest` response is ok in [Line 64](https://github.com/pingcap/pd/pull/1878/files/9f97412617238e1da3ae23ed210127f829dcc4b1#diff-d3014f7ee0630b2f5d78cf7197c9b5e3R62-R65).  But `StatusForbidden` more suitable in here, If the ts used to recovery is smaller than now, we just reject it, it does not mean that the request format is wrong.

Reply:
Seems `Forbidden` is used for auth problems. But I didn't find a proper status code. LGTM.